MonoGame.Framework/Utilities/Lz4Stream/Lz4DecoderStream.cs

```diff
@@ -5,7 +5,7 @@
 using System;
 using System.IO;
 
-namespace Lz4
+namespace Microsoft.Xna.Framework.Content.Pipeline.LZ4
```

Comment:
This namespace should not have `Content.Pipeline` in it as it is not in the content build pipeline.